### PR TITLE
CSS libraries added

### PR DESCRIPTION
**Pull request by noobtuber20103152**
issue #637 

Please review @rupali-codes @CBID2

### CODE REVIEW

File: database/frontend/css-library.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  {
+    "name": "Tailwind CSS",
+    "description": "utility-first CSS framework packed with classes like flex, pt-4, text-center and rotate-90 that can be composed to build any design, directly in your markup.",
+    "url": "https://tailwindcss.com/",
+    "category": "frontend",
+    "subcategory": "css-library"
+  },
+  {
+    "name": "Material UI",
+    "description": "MUI offers a comprehensive suite of UI tools to help you ship new features faster. Start with Material UI, our fully-loaded component library, or bring your own design system to our production-ready components.",
```
Comment (CBID2) [inline database/frontend/css-library.json:18]: [Proofreading]
```suggestion
    "description": "MUI offers a comprehensive suite of UI tools to help you ship new features faster. You can also bring your own design system to its production-ready components.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hey @noobtuber20103152! :) Aside from the proofreading errors I mention, you’re off to a great start! :)

**Review by rupali-codes — CHANGES_REQUESTED** [review]
looks good though, but can you please name it _CSS Libraries_ instead :)

**Comment by noobtuber20103152** [comment]
@rupali-codes @CBID2  done check it again

**Comment by noobtuber20103152** [comment]
done @CBID2

File: database/frontend/css-libraries.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  {
+    "name": "Semantic UI",
+    "description": "Semantic is a development framework that helps create beautiful, responsive layouts using human-friendly HTML.",
+    "url": "https://semantic-ui.com/",
+    "category": "frontend",
+    "subcategory": "css-libraries"
+  },
+  {
+    "name": "Ant Design",
+    "description": "Ant Financial has a large number of enterprise-level products. With complex scenarios, designers and developers often need to respond fast due to frequent changes in product demands and concurrent R&D workflow. Many similar contents exist in the process.",
```
Comment (CBID2) [inline database/frontend/css-libraries.json:32]: [Proofreading] I'm still confused by the last sentence. What is significant about it in regard to developers using this source?
```suggestion
    "description": "Ant Financial has a large number of enterprise-level products. With complex scenarios, designers and developers often need to respond fast due to frequent changes in product demands and concurrent R&D workflow. Many similar contents exist in the process.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @noobtuber20103152! :) Keep on proofreading this PR. There is still some ambiguity in one of your descriptions.

**Pull request closed without merging**